extensions/kafka/src/main/java/com/hazelcast/jet/kafka/KafkaSources.java

```diff
@@ -91,12 +90,13 @@ private KafkaSources() {
      *     restart or failure, the reading continues from the saved offset. You
      *     can achieve exactly-once or at-least-once behavior.
      *
-     *     <li>if processing guarantee is disabled, the source commits the
-     *     offsets to Kafka using {@link KafkaConsumer#commitSync()}. But the
-     *     offsets are committed before or after the event is fully processed.
-     *     Therefore some events can be processed twice or not at all. You can
-     *     configure {@code group.id} in this case. If not configured a random
-     *     UUID will be set.
+     *     <li>if processing guarantee is disabled, the source will start
+     *     reading from default offsets (based on the {@code auto.offset.reset}
+     *     property). You can enable offset committing by assigning a {@code
+     *     group.id}, enabling auto offset committing using {@code
+     *     enable.auto.commit} and configuring {@code auto.commit.interval.ms}
```

Comment:
this is normally on by default, do we explicitly disable it? if so this should be mentioned.

Reply:
We don't disable it, now we do nothing to the properties. It's actually good for us: just by assigning the `group.id` we rely on kafka committing the offsets.